### PR TITLE
Move route to plugin  assigment #2

### PR DESCRIPTION
**Pull request by edwin-bluekite**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -2,29 +2,21 @@
...
 var Hapi = require('hapi');
 var Hoek = require('hoek');
-var Package = require('../package.json');
 
 
 // Declare internals
 
 var internals = {};
-
+internals.version = require('./version');
```
Comment (hueniverse) [inline lib/index.js]: Assign it to `Version`.

File: lib/index.js [hunk 2]
```diff
@@ -2,29 +2,21 @@
...
-
-                return reply({ version: Package.version });
-            }
-        }
+    server.register(internals.version, function (err) {
+	
+	Hoek.assert(!err, err);
     });
 
     server.start(function (err) {
```
Comment (hueniverse) [inline lib/index.js]: `start()` needs to be inside the `register()` callback.

File: lib/version.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            }
+        }
+    });
+
+    return next();
+}
+
+exports.register.attributes = {
+      name: 'version'
+}
```
Comment (hueniverse) [inline lib/version.js]: Missing `;`

**Comment by edwin-bluekite** [comment]
Updated with all suggestions

File: lib/index.js [hunk 4]
```diff
@@ -2,35 +2,24 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
-var Package = require('../package.json');
 
 
-// Declare internals
-
+var Version = require('./version');
```
Comment (hueniverse) [inline lib/index.js]: This should be with the modules section above.

**Pull request closed without merging**